db/migrate/20191021183012_add_odk_style_form_version_number.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class AddOdkStyleFormVersionNumber < ActiveRecord::Migration[5.2]
+  def change
+    add_column :form_versions, :number, :string
+    add_index :form_versions, [:number], unique: true
+
+    reversible do |dir|
```

Comment:
Wow never heard of this before!

Reply:
Seems to be newish, saves a bit of time instead of manually undoing the add_column/index!